core/src/main/java/org/elasticsearch/transport/netty/NettyTransport.java

```diff
@@ -308,6 +312,10 @@ ThreadPool threadPool() {
         return threadPool;
     }
 
+    CircuitBreaker inFlightRequestsBreaker() {
```

Comment:
The circuit breaker limit can be changed dynamically and the circuit breaker service creates a new instance when the setting changes. That's why we always have to get a fresh breaker. Otherwise it would not reflect the changed settings.